RussianFilter/sections/specific.txt

```diff
@@ -49,6 +49,7 @@ crazymailing.com,lotzon.pro,novosti-n.org,prokazan.ru,20minut.ua,alnews.com.ua,g
 ! vast ads on anime sites - for example: animego.org, yummyanime.club, etc
 /:\/\/[0-9]?[a-z0-9]{11,}(\.com|\.in\.net)/$xmlhttprequest,domain=aniqit.com|kodik.info|start.u-cdn.top|ustore.bz|u-stream.in
 !
+sonikelf.ru##nav[class^="widget_text section"]
```

Comment:
Too wide rule. Please exchange it to the `sonikelf.ru#?#.b-sidebar > nav.widget_text div[id^="yandex_rtb"]:upward(nav.widget_text)` -> `css_extended.txt`

Reply:
What's the problem with the rule I created? May I know?

Reply:
As I said above, it's too wide. Can cause false-positives and we avoid such rules.

Reply:
I see.